openapi/2023-12.yaml

```diff
@@ -232,6 +232,32 @@ paths:
           $ref: "#/components/responses/HttpError"
         "5XX":
           $ref: "#/components/responses/HttpError"
+  /unsubscribe:
```

Comment:
Can it be a DELETE endpoint on the `subscriptions` resource?

Reply:
We'd like to keep it expandable and DEL method does not support body parameters. That's the reason for POST.

Reply:
I'll refactor it to DEL /subscriptions?phoneNumber=123

It covers our use case for now.